tests/bears/BearTest.py

```diff
@@ -129,6 +140,24 @@ def test_bad_bear(self):
         # debug message contains custom content, dont test this here
         self.queue.get()
 
+    def test_LocalBear(self):  # Fails because of no run() implementation
```

Comment:
Also the name of the testcase: You don't test the whole `LocalBear`, you test the aspect of printing the filename. Rename it to sth that fits better :+1: